### PR TITLE
Feat/pivot icon with text support

### PR DESCRIPTION
**Pull request by hrestakd**
Pivot items now support icons with text in title. Behaviour is decided via the new optional PivotItem prop linkRenderMode.
If not provided, rendering works same as before.

### CODE REVIEW

File: src/components/Pivot/PivotItem.Props.ts [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import * as React from 'react';
 
+export enum renderModeEnum {
```
Comment (vinkobedek) [inline src/components/Pivot/PivotItem.Props.ts]: how come little r in _renderModeEnum_

Comment (hrestakd) [reply src/components/Pivot/PivotItem.Props.ts]: typo

Comment (hrestakd) [reply src/components/Pivot/PivotItem.Props.ts]: fixed :)

File: src/components/Pivot/Pivot.tsx [hunk 2]
```diff
@@ -117,17 +112,37 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
...
     }
 
+    private _linkRenderer(link: IPivotItemProps): JSX.Element {
+        const { textSize } = this.props;
+        let textStyle;
+        if (textSize) {
+            textStyle = {fontSize: textSize};
+        }
+        const textElement = <span className={'pivot-text'} style={textStyle}>{link.linkText}</span>;
+        const iconElement = <Icon iconName={link.linkIcon} className={'pivot-icon'} title={link.linkText} />;
```
Comment (vinkobedek) [inline src/components/Pivot/Pivot.tsx]: it bugs me a little that an icon element is created if i do not specify a linkIcon. But i will not hold it againts this PR.

**Pull request merged**